ts/features/itwallet/presentation/proximity/machine/tags.ts

```diff
@@ -0,0 +1,3 @@
+export enum ItwPresentationTags {
```

Comment:
We already have `ItwTags` which is a centralized way to manage all tags. Do you think that we could benefit from having a separate enum for proximity machine's tags?